packages/driver/src/cy/commands/aliasing.ts

```diff
@@ -1,18 +1,34 @@
 import _ from 'lodash'
 import $dom from '../../dom'
+import $errUtils from '../../cypress/error_utils'
 
 export default function (Commands, Cypress, cy) {
-  Commands.addQuery('as', function asFn (alias) {
+  Commands.addQuery('as', function asFn (alias, options = {} as Partial<Cypress.AsOptions>) {
```

Comment:
I'd expect to use Cypress.AsOptions directly . Should be able to if the keys are optional.

Reply:
I opted here for consistency. Pre-existing commands all use `Partial<>` in this context, and do not have optional properties.

```
as(alias: string, options?: Partial<AsOptions>): Chainable<Subject>
blur(options?: Partial<BlurOptions>): Chainable<Subject>
check(options?: Partial<CheckOptions>): Chainable<Subject>

...snip...

  interface CheckOptions extends Loggable, Timeoutable, ActionableOptions {
    interval: number
  }
```

Removed the optional-ness of `AsOptions.type` it , since we have Partial<> everywhere AsOptions is used.